### PR TITLE
Bug 1847465: Remove hardcoded CR from plugin

### PR DESCRIPTION
**Pull request by cloudbehl**
- Use example CR to make the request

Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
           </div>
         )}
         {(isIndependent === false || mode === MODES.CONVERGED) && (
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx:123]:
```suggestion
        {(isIndependent === false || mode === MODES.CONVERGED) && csv && (
```
Or else the old bug will start coming

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx [hunk 2]
```diff
@@ -95,25 +107,29 @@ export const CreateOCSServiceForm = withHandlePromise<
...
-    });
+    handlePromise(makeOCSRequest(storageClusterExample, selectedNodes, storageClass, osdSize)).then(
+      () => {
+        dispatch(setFlag(OCS_CONVERGED_FLAG, true));
+        dispatch(setFlag(OCS_FLAG, true));
+        history.push(
+          `/k8s/ns/${ns}/clusterserviceversions/${appName}/${referenceForModel(
+            OCSServiceModel,
+          )}/${OCS_CONVERGED_CR_NAME}`,
+        );
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx:130]: No need to change this.

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx:130]: I just add `storageClusterExample` to makeOCSRequest. and used `OCS_CONVERGED_CR_NAME`.

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx (hunk 1 above)
Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx:123]: Ack

**Pull request closed without merging**